### DISCUSSION TITLE
Alarm like repeating sound?

### DISCUSSION THREAD

**Discussion by crimsonduelist** [poetaman/arttime#59] [General]
Is this supported? Quickly going through issues, wiki I dont see anything
Like if I want 1. set X sound to play let s say a local .m4a file
and 2. loop it for Y amount of time

**Comment by poetaman** [comment]
@crimsonduelist These features aren't yet supported. 1. would make sense to do 2. not as much for various reasons. To begin with, I will add a feature so user can select one of the existing pre-installed sounds on their desktop system. Are you a Linux or macOS or BSD user?

Regarding any specified mp4a/mp3, that change will require change of dependencies. At the moment arttime plays .ogg sound files that come prepackaged on Linux/BSD, and .aiff sounds prepackaged on macOS. If you can find a source of freely licensed mp4a/mp3 sounds that would work well with arttime, I'll be happy to add them to arttime's repository too.

**Comment by crimsonduelist** [comment]
> @crimsonduelist These features aren't yet supported. 1. would make sense to do 2. not as much for various reasons. To begin with, I will add a feature so user can select one of the existing pre-installed sounds on their desktop system. Are you a Linux or macOS or BSD user?

On linux. What I m trying to do is use this as an alarm. I m looking through the wiki to see how to make a keypoem that would match my alarms. So I could basically just keep updating that when I needed to add an alarm for example.

My point for 2. is if u wanna use this as an alarm a notification like sound isnt really cutting it no? 
Ofcourse no point making it loop for a "custom" interval I should ve worded that better originally, but shouldnt it keep going until u stop it for example?

> Regarding any specified mp4a/mp3, that change will require change of dependencies. At the moment arttime plays .ogg sound files that come prepackaged on Linux/BSD

U mean from /usr/share/sounds/ ? and also not specifically .ogg but ogg container format? so .oga would still work. 
I m assuming that is the case since I dont see any .ogg files searching on my arch install, only .oga but still have notification sounds.
I ll try to look for a library for mp4a/mp3 also. I ll see what other alarm/alarm-like apps re using I guess
But I m not that hung up on the format for now so long as it works as I need it.

> To begin with, I will add a feature so user can select one of the existing pre-installed sounds on their desktop system. 

Btw how do you mean? like if I wanted to specify a goal of 2 seconds I could add a sound to it like that "2s sound.ogg" ?
